### ISSUE TITLE
Add Mozilla GitHub projects as well

### ISSUE THREAD

**Issue by ngsankha** [MozillaIndia/leaderboard#9]
New projects like Rust, Servo, Persona are on GitHub. Use the GitHub API to access them as well.

**Comment by sudheesh001** [comment]
How would you like this to be done. Using the github.js or the gh3.js with a dependency on jQuery, base64 and Underscore ?

In github.js, we have

``` html
<script src="lib/underscore-min.js">
<script src="lib/base64.js">
<script src="github.js">
```

``` javascript
var user = github.getUser(); 
//Gives list of all user repos, private, public and those contributed to.
```
1. How about keeping all the mozilla projects from https://github.com/mozilla/ into an array ? i.e. 616 repos (BIG!, Need something better) 
2. Keep a json file for the list of projects and let each user who contributes add the project extension i,e

``` json
// tree, repo
// Translated as https://github.com/<tree>/<repo>
var users = [
 ["mozilla", "kitsune"],
 ["mozilla", "socorro"],
 ["mozilla", "rust"],
 ["mozilla", "servo"],
 ["jdm","asknot"],
];
```

Once this is done, easiest would be to access list of contributors in specifc project from the project's **network/members** or the **/contributors** it also gives the number of commits made. Maybe thats of help ?

**Comment by grssam** [comment]
Fixed by #37
